library/ws-widget/collection/src/lib/_common/profile-card-stats/profile-card-stats.component.ts

```diff
@@ -72,7 +72,7 @@ export class ProfileCardStatsComponent implements OnInit {
       const defaultData = this.configSvc.profileTimelyNudges.data[this.configSvc.profileTimelyNudges.data.length - 1]
       if (defaultData) {
         this.republicDayData['backgroupImage'] = defaultData.backgroupImage
-        this.republicDayData['info'] = defaultData['info'][rand]
+        this.republicDayData['info'] = defaultData['webInfo'][rand]
```

Comment:
Please check , did you made changes in respective json file for "webInfo" and "webGreet"

Reply:
yes, They are already merged to 11.1
Mobile they use info object and webInfo object for web